modules/web-cli/assets/js/modules/component-base.js

```diff
@@ -392,6 +391,29 @@ export default class ComponentBase extends Module {
 			.addClass( 'elementor-active' );
 	}
 
+	activateControl( controlPath ) {
+		if ( ! controlPath ) {
+			return;
+		}
+
+		const editor = elementor.getPanelView().getCurrentPageView();
+		const currentView = editor.content ? editor.content.currentView : editor;
+
+		const controlView = this.getControlViewByPath( currentView, controlPath );
+		controlView.activate?.();
+	}
+
+	getControlViewByPath( currentView, controlPath ) {
+		const controls = controlPath.split( '/' );
+		let controlView = currentView.getControlViewByName( controls[ 0 ] );
+
+		controls.slice( 1 ).forEach( ( control ) => {
+			controlView = controlView.getChildControlView?.( control );
+		} );
+
+		return controlView;
+	}
```

Comment:
It shouldn't be here, web-cli is an independent module

Reply:
Can you please resolve this @matipojo ?